classes/01class/exercises/c01-e04/marcio-anatrielo/c01-e04/c01-e04.txt

```diff
@@ -0,0 +1,65 @@
+
+
+Create S3 buckets 
+
+ devops-academu-au
+
+Properties
+ Events 0 Active notifications
+ Versioning Disabled
+ MFA delete Disabled
+ Logging Disabled
+ Static web hosting Disabled
+  Tags 1 Tags
+ Requester pays Disabled
+ Object lock Disabled
+ Transfer acceleration Disabled
+Permissions
+ Owner marcio.anatrielo
+ Block public access Enabled
+ Bucket policy No
+ Access control list 1 Grantees
+ CORS configuration No
+Management
+ Lifecycle Disabled
+ Replication Disabled
+ Analytics Disabled
+ Inventory Disabled
+ Metrics Disabled
+
+
+Create IAM 
+	
+
+ marcioanatrielo 
+
+User ARN
+arn:aws:iam::650909960884:user/marcioanatrielo 
+Path
+/
+Creation time
+2020-02-09 16:09 UTC+1100
+
+User permissions
+
+ Administrator Access
+ PowerUserAccess
+
+Access key ID AKIAZPDKBZK2N4LR5JI2
```

Comment:
remove your access key from this file.. make sure to delete  this key from your account as well as  now it has  been  exposed.

Reply:
Can you explain if you manually configured your access keys on your instance or you used the instance role to get access to S3? I create this manually, just copy and past  on my VM and change permission with chmod to 644 to have access to run ssh.

Reply:
remove your access key from this file.. make sure to delete this key from your account as well as now it has been exposed. - I remove all key and machine already and in this file is note available my password just key ID with I believe is similar a user name is that correct?